src/components/content/ProjectsContent.jsx

```diff
@@ -0,0 +1,167 @@
+import { createUseStyles } from "react-jss";
+import { Link } from "react-router-dom";
+import edit from "../../assets/edit.svg";
+import deleteicon from "../../assets/delete.svg";
+
+function ProjectsContent() {
+  const classes = useStyles();
+
+  // Sample project data
+  const projects = [
+    {
+      name: "Project 1",
+      creationDate: "2022-01-01",
+    },
+    {
+      name: "Project 2",
+      creationDate: "2022-01-02",
+    },
+    {
+      name: "Project 3",
+      creationDate: "2022-01-03",
+    },
+  ];
+
+  return (
+    <div className={classes.body}>
+      <h1>Projects</h1>
+      <Link to="/create-project" className={classes.button}>
```

Comment:
Hi Jake,
The code appears to be in good shape, but there's an issue with it causing the site to crash when clicking on "Create Project," which is less than ideal. After debugging, I discovered that the problem lies with the "Link" tag used for "Create Project." To resolve this, please replace the "Link" tag with a button and remove the "href" attribute ("/create-project"). I'll provide the necessary changes below. Thank you.
![chrome_ny8SIz4ac2](https://github.com/BIT-Studio-OP/CraftMyPortfolio/assets/81060521/26d01185-0335-4dc4-8558-06df9f0426eb)

File path: src/components/content/ProjectsContent.jsx

From this :
![image](https://github.com/BIT-Studio-OP/CraftMyPortfolio/assets/81060521/a60e2eae-9167-4ce3-90ae-52fed19c85e9)
To this:
![image](https://github.com/BIT-Studio-OP/CraftMyPortfolio/assets/81060521/2c3c9783-0281-487d-a2f5-ca2254d1dfe7)